src/super_gradients/training/sg_trainer/sg_trainer.py

```diff
@@ -2432,72 +2435,18 @@ def quantize_from_config(cls, cfg: Union[DictConfig, dict]) -> QuantizationResul
         :raises ImportError: If pytorch-quantization import was unsuccessful
 
         """
-        import_pytorch_quantization_or_install()
-
         # INSTANTIATE ALL OBJECTS IN CFG
         cfg = hydra.utils.instantiate(cfg)
 
-        # TRIGGER CFG MODIFYING CALLBACKS
-        cfg = cls._trigger_cfg_modifying_callbacks(cfg)
-
-        quantization_params = get_param(cfg, "quantization_params")
-        if quantization_params is None:
-            logger.warning("Your recipe does not include quantization_params. Using default quantization params.")
-            quantization_params = load_recipe("quantization_params/default_quantization_params").quantization_params
-            cfg.quantization_params = quantization_params
-
-        export_params = get_param(cfg, "export_params", {})
-        export_params = ExportParams(**export_params)
-
         if get_param(cfg.checkpoint_params, "checkpoint_path") is None and get_param(cfg.checkpoint_params, "pretrained_weights") is None:
-            raise ValueError("Starting checkpoint / pretrained weights are a must for QAT finetuning.")
+            raise ValueError("Starting checkpoint / pretrained weights are a must for quantization.")
 
+        # TODO: Should this really be here?
```

Comment:
Why not?
We can do PTQ/QAT on multiple GPUs (I specifically remember I wrote support for it in the past...)

Reply:
Okie, removing todo then)